packages/firebase_auth/firebase_auth_web/lib/src/firebase_auth_web_recaptcha_verifier_factory.dart

```diff
@@ -61,24 +61,25 @@ class RecaptchaVerifierFactoryWeb extends RecaptchaVerifierFactoryPlatform {
 
     if (container == null || container.isEmpty) {
       parameters['size'] = 'invisible';
-      Element /*?*/ el = window.document.getElementById(_kInvisibleElementId);
+      Element? el = window.document.getElementById(_kInvisibleElementId);
 
       // If an existing element exists, something may have already been rendered.
       if (el != null) {
         el.remove();
       }
 
-      window.document.documentElement.children
+      window.document.documentElement!.children
           .add(DivElement()..id = _kInvisibleElementId);
 
       element = _kInvisibleElementId;
     } else {
       parameters['size'] = convertRecaptchaVerifierSize(size);
       parameters['theme'] = convertRecaptchaVerifierTheme(theme);
 
-      Element el = window.document.getElementById(container);
```

Comment:
It didn't feel correct to do nothing with `el` besides an assertion. This change no-longer calls `getElementById` when the asserts are removed

Reply:
Agree 👍